doc/systest_basics.rst

```diff
@@ -0,0 +1,297 @@
+
+Shear-shear correlation functions are :math:`\xi_+` and :math:`\xi_-`.  They are functions of the
+spatial correlation of the shears and their complex conjugates in sky coordinates; see e.g.
+`Jarvis et al. 2016 <http://adsabs.harvard.edu/abs/2016MNRAS.460.2245J>`_ for complete definitions
+of :math:`\xi_+` and :math:`\xi_-`.  The :math:`\xi` functions should be completely real (within 
+noise), although the results will include the imaginary portions as a cross-check.
+
+Point-scalar (point-kappa) estimates are equivalent to ``<scalar>``. Scalar-shear estimates are
+equivalent to ``<scalar*Re(shear)>`` with a corresponding imaginary case ``<scalar*Im(shear)>``,
+corresponding to the correlation between the scalar value of the first sample at that point and the
+tangential shear (real) or the shear at 45 degrees from the tangent (imaginary) from the
+second sample. Scalar-scalar estimates are ``<scalar1*scalar2>``.  Random catalogs result in
+compensated estimators as in the point-shear case.
+
+Whisker plots
+-------------
+
+Whisker plots are child classes of :class:`stile.sys_tests.BaseWhiskerPlotSysTest`.  They're the
+standard weak lensing whisker plots: visualizations of the shear field using headless arrows.
+The predefined types are:
+
+- **Star**: whisker plot of shapes of PSF stars
+- **PSF**: whisker plot of PSF shapes at the location of PSF stars
+- **Residual**: whisker plot of (star shape-PSF shape)
+- **None**: an empty :class:`stile.sys_tests.BaseWhiskerPlotSysTest` class instance, which can be
+  used for multiple types of whisker plots.  See the documentation for
+  :class:`stile.sys_tests.BaseWhiskerPlotSysTest` (especially the method
+  :func:`whiskerPlot <stile.sys_tests.BaseWhiskerPlotSysTest.whiskerPlot>`) for more details.
+  Note that this type has a different call signature than
+  the other methods and that it lacks many of the convenience variables the other
+  WhiskerPlots have, such as self.objects_list and self.required_quantities.
+
+Calls to child classes of :class:`stile.sys_tests.BaseWhiskerPlotSysTest`
+return :class:`matplotlib.figure.Figure` instances and can be saved directly as images, but
+a :func:`.plot` method is included (that just returns the figure passed to it) so the plotting
+interface is the same as other :class:`SysTests <stile.sys_tests.SysTest>`.
+
+Scatter plots
+-------------
+
+Scatter plots are child classes of :class:`stile.sys_tests.BaseScatterPlotSysTest`.  They generate
+scatter plots of the data plus a linear regression fit through the scattered points.  There is
+also an optional comparison trendline at :math:`y=0` or :math:`x=y` (or a user-defined function).
+The predefined types are:
+
+- **StarVsPSFG1**: star vs PSF g1
+- **StarVsPSFG2**: star vs PSF g2
+- **StarVsPSFSigma**: star vs PSF sigma
+- **ResidualVsPSFG1**: (star - PSF) g1 vs PSF g1
+- **ResidualVsPSFG2**: (star - PSF) g1 vs PSF g2
+- **ResidualVsPSFSigma**: (star - PSF) g1 vs PSF sigma
```

Comment:
I'm a little confused why these are all g1 residual vs. (other stuff) but e.g. we don't have g2 residual vs. g2 or sigma residual vs. sigma.  Is that a typo?